event/handler.go

```diff
@@ -83,6 +86,14 @@ func (h *ImagePublishedHandler) Handle(ctx context.Context, event *ImagePublishe
 	}
 	log.Event(ctx, "event handler called", log.INFO, logData)
 
+	// GET images/{id}/downloads/{variant}
+	imageDownload, err := h.ImageAPICli.GetDownloadVariant(ctx, "", h.AuthToken, "", event.ImageID, event.ImageVariant)
+	if err != nil {
+		log.Event(ctx, "error getting image variant from API", log.ERROR, log.Error(err), logData)
+		h.setImageStatusToFailed(ctx, event.ImageID, "error getting image variant from API")
```

Comment:
I'd probably add more context to the error text, eg.
```suggestion
		h.setImageStatusToFailed(ctx, event.ImageID, fmt.Sprintf("error getting image variant '%s' from API", event.ImageVariant))
```